sessions/js-structure/quiz-app/components/header/header.css

```diff
@@ -0,0 +1,14 @@
+.header {
+  position: fixed;
+  z-index: 20;
```

Comment:
If `20` is meant to signify an "always on top" value, I'd probably pick a significantly higher number. 🤔

Reply:
Since it's the only place where the z-index is used, I would leave it this way here.